01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ruby
+(1..20).each do |x|
+  if (x%3 == 0 && x%5 != 0)
```

Comment:
演算子の前後には半角スペースを空けた方が良いです。一般的な慣習ですし、のちに出てくるlinter(rubocop)にも指摘されると思いますので今のうちに習慣づけておきましょう。

```suggestion
  if (x % 3 == 0 && x % 5 != 0)
```

Reply:
あと、rubyなので`if`の括弧はなくてもいいかもです。
```suggestion
  if x % 3 == 0 && x % 5 != 0
```
あってもいいんですが、大抵の人は括弧をつけていない気がしますね。タイプ数も減ります。